Makefile

```
@@ -6,7 +6,7 @@ CARGO_HOME := $(TOP).cargo
 VERSION := $(shell awk '/^release-version = /{ print $$3 }' $(TOP)Twoliter.toml)
 GIT_HASH := $(shell git describe --always --dirty --exclude '*' || echo 00000000 )
 
-TWOLITER_VERSION ?= "0.3.0"
+TWOLITER_VERSION ?= "0.4.0"
```

Are we bumping again?

```suggestion
TWOLITER_VERSION ?= "0.5.0"
```